src/server/game/Scripting/ScriptMgr.cpp

```diff
@@ -96,6 +96,10 @@ template<>
 struct is_script_database_bound<AchievementCriteriaScript>
     : std::true_type { };
 
+template<>
+struct is_script_database_bound<AreaTriggerEntityScript>
```

Comment:
Thanks for review, i will take a look :)

I suppose i have to do the same thing for SceneScript ?

https://github.com/TrinityCore/TrinityCore/pull/17976